### ISSUE TITLE
Padding wrongly computed in layers with striding when converting for *.pb file

### ISSUE THREAD

**Issue by phager90** [onnx/tensorflow-onnx#1028]
**Describe the bug**
The padding is not correctly computed on layers with strides when converting from *.pb files.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 20.04.1 LTS
- Tensorflow Version: 2.3.0
- Python version: 3.6.10
- tf2onnx: 1.7.0

**To Reproduce**
Download mobilenet graphdef from google
https://storage.googleapis.com/mobilenet_v2/checkpoints/mobilenet_v2_1.0_224.tgz
Convert with:
`python -m tf2onnx.convert --graphdef ./mobilenet_v2_1.0_224_frozen.pb  --output onnx_3x224x224.onnx --inputs input:0[1,224,224,3] --inputs-as-nchw input:0  --outputs MobilenetV2/Logits/Squeeze:0`

**Expected behavior**
Proper computation of padding in layers with strides:
In output onnx file (onnx_3x224x224.onnx) in layer **MobilenetV2/Conv/BatchNorm/FusedBatchNorm** padding is expected to be 1,1,1,1 and not 0,0,1,1

**Screenshots**

pb file:
![image](https://user-images.githubusercontent.com/9904169/88960488-def01680-d2a3-11ea-9466-65ff19628e15.png)

onnx file:
![image](https://user-images.githubusercontent.com/9904169/88960515-eb746f00-d2a3-11ea-8816-3a25a88dba8d.png)

**Comment by guschmue** [comment]
The pads in onnx are in the format [x1_begin, x2_begin...x1_end, x2_end]
https://github.com/onnx/onnx/blob/master/docs/Operators.md#conv

**Comment by TomWildenhain-Microsoft** [comment]
Is there a different reason you believe this model to be converting incorrectly?

**Comment by phager90** [comment]
Hi Tom,
I checked this again. 
I was doing a 1:1 comparison of the exported ONNX file of the same network (mobilenet) once exported from Pytorch and once exported from Tensorflow. I was expecting to get the same convolution arguments for all operations.

I did not thought about the padding discrepancy between TF and Pytorch (#4 ):

![image](https://user-images.githubusercontent.com/9904169/89383230-6c949180-d6fc-11ea-83a9-4b122e4870f8.png)

I was assuming pytorch-style padding and thus was expecting the padding to be [1,1,1,1]

Given that the onnx file should play back during runtime the same way as the model did in TF, I agree that the [0,0,1,1] padding is in fact correct.

Please excuse my mistake.
